src/PrestaShopBundle/Translation/Translator.php

```diff
@@ -29,6 +29,9 @@
 
 use Symfony\Bundle\FrameworkBundle\Translation\Translator as BaseTranslator;
 
+/**
+ * Replaces the original Symfony FrameworkBundle translator
```

Comment:
I'm not sure this PHPDoc is true, it's the compiler pass that do the replacement.

Reply:
I didn't think it could be interpreted that way. Here's a better wording:

```suggestion
 * Replacement for the original Symfony FrameworkBundle translator
```